spec/intl.html

```diff
@@ -1338,6 +1338,259 @@ <h1>CalendarEraYear ( _calendar_, _dateLike_ )</h1>
             1. Return _result_.
           </emu-alg>
         </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatetoiso" type="abstract operation">
+          <h1>
+            CalendarDateToISO (
+              _calendar_: a String,
+              _fields_: a Record,
+              _overflow_: an Object,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It performs implementation-defined processing to convert _fields_ that describes a date in the calendar represented by the string _calendar_ to the ISO calendar and returns the corresponding day, month and year values as a Record.</dd>
+          </dl>
```

Comment:
I think we should add 
```
<emu-alg>
        1. Let _requestedLocales_ be ? CanonicalizeLocaleList(_locales_).
        1. Let _isoYear_ be the result of performing implementation-defined process to convert _fields_ that describes a date in the calendar represented by the string _calendar_ to the year in ISO calendar.
        1. Let _isoMonth_ be the result of performing implementation-defined process to convert _fields_ that describes a date in the calendar represented by the string _calendar_ to the month in ISO calendar.
        1. Let _isoDay_ be the result of performing implementation-defined process to convert _fields_ that describes a date in the calendar represented by the string _calendar_ to the day in ISO calendar.        
        1. Record { [[Year]]: _isoYear_, [[Month]]: _isoMonth_, [[Week]]: _isoDay_ }.
</emu-alg>
```
we need to clearly states which fields the Record used as key in order to make it a tightly defined AO. Same as all aO below.

Reply:
I don't think it's necessary to define it with algorithm steps like this, but the comment seems to have been addressed with the new description of this AO.